_includes/langselect.html

```diff
@@ -11,7 +11,7 @@
         {% else %}
           {% capture langlink %}/{{ lng }}{{ page.url }}{% endcapture %}
           {% capture lang_rel %}alternate{% endcapture %}
-          {% endif %}
+        {% endif %}
           <li><a href="{{ langlink }}" rel="{{ lang_rel }}" data-slang="{{ lng }}">{{ lng }}</a></li>
```

Comment:
This needs two spaces off - line 6 is 8 spaces, this should be the same level.

Reply:
Sorry, not too familiar with this - is that right?

Reply:
Don't worry, spacing is wrong all over the place - not worth addressing as part of this change.